FreeType.xs

```diff
@@ -144,6 +144,9 @@ const static QefFT2_Uv_Const qefft2_uv_const[] =
     QEFFT2_CONSTANT(FT_LOAD_LINEAR_DESIGN)
 #if QEFFT2_FT_AT_LEAST(2,1,3)
     QEFFT2_CONSTANT(FT_LOAD_NO_AUTOHINT)
+#endif
+#if QEFFT2_FT_AT_LEAST(2,6,1)
+    QEFFT2_CONSTANT(FT_LOAD_COMPUTE_METRICS)
```

Comment:
If u export constant it needs to be documented/mentioned in `Freetype.pm`, as the other constants.